update: libcurl tutorial (#4783)
JetBrains/kotlin-web-site: docs/topics/native/native-app-with-c-and-libcurl.md

```

As you can see, explicit variable declarations are eliminated in the Kotlin version, but everything else is pretty much
the same as the C version. All the calls you'd expect in the libcurl library are available in the Kotlin equivalent.

> This is a line-by-line literal translation. You could also write this in a more Kotlin idiomatic way.
>
{type="tip"}

## Compile and run the application

> 1. Compile the application. To do that, call `runDebugExecutableNative` in the list of run Gradle tasks or use the following

Reviewer: ```suggestion
1. Compile the application. To do that, run the `runDebugExecutableNative` Gradle task from the list of available tasks or use the following
```
